flutter/news/lib/core/error/failures.dart

```diff
@@ -5,4 +5,16 @@ abstract class Failure extends Equatable {
 
   @override
   List<Object> get props => const [];
```

Comment:
I wonder about this, if I understand correctly this means that a `Failure` might contain a list of properties, and properties are of any type.

Then I see t hat actually we only have `message` which is a string for the failure.

If at the moment we are only using `message` as the field contained by a failure, why not making this be the property that a `failure` might have, instead of a list of any object? 

We can always change and adapt to our needs, as they happen

Reply:
updated on https://github.com/novoda/multiplatform-playground/commit/8323a52c3c021a85d1662fa2609c061534a5fce3